tests/integration/features/chat/notifications.feature

```diff
@@ -0,0 +1,216 @@
+Feature: chat/notifications
+
+  Background:
+    Given user "participant1" exists
+    Given user "participant2" exists
+
+  Scenario: Normal message when recipient is online in the one-to-one
+    When user "participant1" creates room "one-to-one room" (v4)
+      | roomType | 1 |
+      | invite   | participant2 |
+    Given user "participant2" joins room "one-to-one room" with 200 (v4)
```

Comment:
I would say that everything before sending the message should be a precondition, and therefore the steps should be prefixed by `Given` and `And` (here and in all the scenarios of this file), but I guess that is just a matter of taste :-)